src/components/dashboard/content/list/FaultTreeAndSystemOverviewTable.tsx

```diff
@@ -5,10 +5,11 @@ import { useTranslation } from "react-i18next";
 import useStyles from "./FaultTreeOverviewTable.styles";
 import { FaultTree } from "@models/faultTreeModel";
 import { useNavigate } from "react-router-dom";
-import { ROUTES, SELECTED_SYSTEM } from "@utils/constants";
+import {ROUTES} from "@utils/constants";
```

Comment:
not going over prettier

Reply:
I think there is an issue with Prettier in this repo... I have to trigger it manually (even if my settings show that it should be on every save + we are supposed to have a pre-commit hook that prevents these issues)